dialect/sql/schema/migrate.go

```diff
@@ -208,7 +208,7 @@ func (m *Migrate) txCreate(ctx context.Context, tx dialect.Tx, tables ...*Table)
 			}
 			// indexes.
 			for _, idx := range t.Indexes {
-				query, args := m.addIndex(idx, t.Name).Query()
+				query, args := m.addIndex(idx, t.Name).IfNotExists().Query()
```

Comment:
Can you please move this configuration to `sqlite.go` and `postgres.go` as it's not supported by MySQL/MariaDB.

Reply:
Done.